service/push_notification.py

```diff
@@ -107,10 +111,10 @@ def send_notifications(self):
         if 'firebase_token' in selected_calendar.keys():
             results = push_service.notify_single_device(
                 registration_id=selected_calendar['firebase_token'],
-                message_body="success")
-            result = {}
+                message_body="success")  
+            result = OrderedDict()
             result['results'] = results['results']
-            result['subscriber_info'] = selected_calendar['firebase_token']
```

Comment:
@RachaelNantale what happens to the firebase_token key/value being used here? Is it no longer relevant?

Reply:
Thank you @joshuaocero  for the feedback, it is still relevant and I have re-added it.